src/helpers/fs-functions.ts

```diff
@@ -127,14 +171,23 @@ export const fetchAllActivityPosts = async (count: number): Promise<{ batch: Hom
   const currentMonth = monthsInOrder[startMonth];
 
   const allpossibleMonths = createMonthsFromKeys(startYear, endYear);
-  const sliceIndex = allpossibleMonths.findIndex((month) => month.slice(0, -5) === currentMonth);
+  const sliceIndex = allpossibleMonths.findIndex(
+    (month) => month.slice(0, -5) === currentMonth
+  );
 
   let monthsToBegin = allpossibleMonths.slice(sliceIndex);
 
-  const LTDIRECTORY = `${process.cwd()}/public/static/static/${folders[0]}/${monthsToBegin[count]}`;
-  const BTDIRECTORY = `${process.cwd()}/public/static/static/${folders[1]}/${monthsToBegin[count]}`;
+  const LTDIRECTORY = `${process.cwd()}/public/static/static/${folders[0]}/${
+    monthsToBegin[count]
+  }`;
+  const BTDIRECTORY = `${process.cwd()}/public/static/static/${folders[1]}/${
+    monthsToBegin[count]
+  }`;
 
-  const isFolderPresent = fs.existsSync(LTDIRECTORY) || fs.existsSync(BTDIRECTORY);
+  const LIGHTNINGLEGACY =  ['April_2024', 'March_2024', 'Feb_2024','Jan_2024', 'Dec_2023', 'Nov_2023' ]
```

Comment:
What is special about these months? I thought the lightning-dev mailing list went further back than this but maybe I'm wrong?